tests/python_package_test/test_engine.py

```diff
@@ -2261,9 +2261,8 @@ def test_forced_bins():
     x[:, 0] = np.arange(0, 1, 0.01)
     x[:, 1] = -np.arange(0, 1, 0.01)
     y = np.arange(0, 1, 0.01)
-    forcedbins_filename = str(
-        Path(__file__).absolute().parents[2] / 'examples' / 'regression' / 'forced_bins.json'
-    )
+    forcedbins_filename = (Path(__file__).absolute().parents[2] / 'examples'
+                           / 'regression' / 'forced_bins.json')
```

Comment:
I think the original indentation currently on `master` is easier to read than this (here and on the similar one below). Would you consider just removing the `str()` but otherwise leaving the original indentation?

Reply:
Yes, agree with you! I don't know why I chose to change indentation 🙂 .

Reply:
```suggestion
    forcedbins_filename = (
        Path(__file__).absolute().parents[2] / 'examples' / 'regression' / 'forced_bins.json'
    )
```